packages/pwa-buildpack/lib/Utilities/graphQL.js

```diff
@@ -45,6 +45,8 @@ const getSchemaTypes = () => {
 };
 
 /**
+ * @deprecated use getPossibleTypes with ApolloClient v3.
```

Comment:
@jcalcaben how do I link `getPossibleTypes` to the function in this file?

Reply:
```
{@link getPossibleTypes}
```

Reply:
```suggestion
 * @deprecated use {@link getPossibleTypes} with ApolloClient v3.
```